File: app/src/test/java/com/tomaszrykala/dorefindmi/things/supplier/abcleds/MockAbcLedsSupplier.java
package com.tomaszrykala.dorefindmi.things.supplier.abcleds;

public class MockAbcLedsSupplier implements AbcLedsSupplier {

    @Override public void init() {
        System.out.println("MockAbcLedsSupplier.init");
    }

    @Override public void close() throws Exception {
        System.out.println("MockAbcLedsSupplier.close");
    }

    @Override public void lightFor(boolean onPadA, boolean onPadB, boolean onPadC) {
        System.out.print("MockAbcLedsSupplier.lightFor; ");
        System.out.println("onPadA = [" + onPadA + "], onPadB = [" + onPadB + "], onPadC = [" + onPadC + "]");
    }
}

File: app/src/test/java/com/tomaszrykala/dorefindmi/things/supplier/abcleds/MockAbcLedsSupplier.kt
package com.tomaszrykala.dorefindmi.things.supplier.abcleds

class MockAbcLedsSupplier : AbcLedsSupplier {

    @Throws(Exception::class)
    override fun close() {
        println("MockAbcLedsSupplier.close")
    }

    override fun lightFor(onPadA: Boolean, onPadB: Boolean, onPadC: Boolean) {
        print("MockAbcLedsSupplier.lightFor; ")
        println("onPadA = [$onPadA], onPadB = [$onPadB], onPadC = [$onPadC]")
    }
}
